tokenomics/mkr-tokenomics.md

```diff
@@ -14,7 +14,13 @@ There are a multiple goals that Maker intends to meet via the new MKR tokenomics
 2. To link the value and success of Maker Core with that of its SubDAOs and vice versa.
 3. To incentivize talented individuals to work as part of Maker Core.
 
-## Endgame MKR Emissions
+## MKR Utility
+
+MKR may now be used to borrow Dai by holders that have delegated their governance power to a [Delegate](../maker-core/delegates.md). This is made possible through the [Easy Governance Frontend](../maker-core/easy-governance-frontend.md).
+
+SubDAO tokens will also be farmable using delegated MKR.
+
```

Comment:
Add a link to upcoming Sagittarius Engine page here? It doesn't have too many additional details now but probably will in the future